machine/wnc_x86_64/rootconf/sysroot-lib-onie/init-platform

```diff
@@ -0,0 +1,30 @@
+# -*- shell-script -*-
+
+#  Copyright (C) 2014 Curt Brune <curt@cumulusnetworks.com>
+#
+#  SPDX-License-Identifier:     GPL-2.0
+
+# Demonstration of the init-platform functionality
+
+# This script fragment is sourced by /etc/init.d/init-arch.sh, which
+# in turn executes the init_platform_pre_arch() and
+# init_platform_post_arch() functions.
+
+# A machine can use this feature to run any early boot scripts needed
+# by the machine.  A typical usage would be to initialize specific
+# hardware devices.
+
+# Use this function to perform any initializations required by the
+# architecture specific initialization.  This function executes before
+# the architecture initializations.
+init_platform_pre_arch()
+{
+    echo "Running demonstration platform init pre_arch routines..."
+}
+
+# Use this function to perform any post-initializations or clean-up.
+# This function executes after the architecture initializations.
+init_platform_post_arch()
+{
+    echo "Running demonstration platform init post_arch routines..."
+}
```

Comment:
Echo above comments about copying from the example machine.  If you do not need it, do not include it.